src/pages/BrowserPage.qml

```diff
@@ -182,12 +182,21 @@ Page {
         height: inputMask.height
     }
 
-    Rectangle {
+    ShaderEffect {
```

Comment:
DimmerEffect component maybe that could be drop to components folder.

public properties could be like...

```
property color color: "white"
property real alpha: 1.0
readonly property color dimmerColor: Theme.rgba(color, alpha < 0.0 ? 0.0 : alpha > 1.0 ? 1.0 : alpha)

blending: false
visible: alpha > 0.0
...
```

Over here would look like:

```
Browser.DimmerEffect {
    id: contentDimmer
    width: browserPage.width
    height: Math.ceil(overlay.y)
    alpha: 0.9 - (overlay.y / (webView.fullscreenHeight - overlay.toolBar.toolsHeight)) * 0.9
    color: Theme.highlightDimmerColor
}
```

Then I think that this could be reused in browser.qml, OrientationFader, and BrowserContextMenu.